File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex12.java
//Importando Scanner
import java.util.Scanner;

public class Ex12 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Terços de 3 Vetores\n\n");
		
		//vars
		int a[] = new int[9];
		int b[] = new int[9];
		int c[] = new int[9];
		int res[] = new int[9];
		int i;
		
		//Entradas
		for( i=0; i<9; i++)
		{
			System.out.print("Digite a[" + i + "]: ");
			a[i] = input.nextInt();
		}
		
		for( i=0; i<9; i++)
		{
			System.out.print("Digite b[" + i + "]: ");
			b[i] = input.nextInt();
		}
		
		for( i=0; i<9; i++)
		{
			System.out.print("Digite c[" + i + "]: ");
			c[i] = input.nextInt();
		}
		
		//Processamento
		for(i=0; i<3; i++)
		{
			res[i] = a[i];
		}
		
		for(i=3; i<6; i++)
		{
			res[i] = b[i];
		}
		
		for(i=6; i<9; i++)
		{
			res[i] = c[i];
		}
		
		//Saída
		System.out.println("\n\nVetor Resultante: ");
		for( i=0; i<9; i++)
		{
			System.out.printf("%d  ", res[i]);
		}
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex12.c
#include <stdio.h>

main()
{
    printf("Juntando tercos de vetores\n\n");

    //vars
    int v1[9], v2[9], v3[9], res[9], i;

    //Entradas
    for( i=0; i<9; i++)
    {
        printf("Digite v1[%d]: ", i);
        scanf("%d", &v1[i]);
    }

    for( i=0; i<9; i++)
    {
        printf("Digite v2[%d]: ", i);
        scanf("%d", &v2[i]);
    }

    for( i=0; i<9; i++)
    {
        printf("Digite v3[%d]: ", i);
        scanf("%d", &v3[i]);
    }

    //Processamento
    for( i=0; i<3; i++)
    {
        res[i] = v1[i];
    }

    for( i=3; i<6; i++)
    {
        res[i] = v2[i];
    }

    for( i=6; i<9; i++)
    {
        res[i] = v3[i];
    }

    //Saída
    printf("\n\nVetor Resultante: \n");
    for( i=0; i<9; i++)
    {
        printf("%d  ", res[i]);
    }

}
